src/radiant.c

```diff
@@ -689,12 +695,14 @@ int radiant_read_event(radiant_dev_t * bd, rno_g_header_t * hd, rno_g_waveform_t
   wf->run_number = bd->run; 
 
   wf->radiant_nsamples = nsamples;
-  
-  radiant_get_delays(bd,&wf->radiant_readout_delays);
+  radiant_readout_delay_t rdout;
```

Comment:
do you want to initialize to 0?  you an do `radiant_readout_delay_t rdout = {0};` to do that.

Reply:
actually, you can just access directly from bd since you're inside radiant.c and have access to its definition.

Reply:
there's a little translation that goes on in radiant_get_delays to go from the group mask (which is physically set in fpga memory) to a channel mask. I think I left it in the function to be organized. though not so useful (again) and can be replaced with just a simple read from the radiant device

Reply:
and whenever the delays get set, they can get translated to channel mask since that's how I have it in the radiant device struct